CalcualtorLogic.js

```diff
@@ -14,7 +13,6 @@ class CalculatorLogic {
     };
     divide(number1, number2) {
         let result = number1 / number2;
-        if (number2 === 0) result.textContent = 'Cannot divide by zero';
```

Comment:
You have removed this line which used to handle the divide by zero issue 
how you're handling divide by zero now?
I only asked you about the **result.textContent** 
is `result` an object or something else?

Reply:
I returned it back before seeing your comment
originally we do not have something called result in the oop cal pro
we only have current result 
so result.texContent just some errors 
not object and not string and not Boolean and not number ..etc